services/ui-waflog-s3-bucket/serverless.yml

```diff
@@ -0,0 +1,30 @@
+service: ui-waflog-s3-bucket
+
+frameworkVersion: "2"
+
+plugins:
+  - serverless-stack-termination-protection
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  region: us-east-1
+
+custom:
+  stage: ${opt:stage, self:provider.stage}
+  serverlessTerminationProtection:
+    stages:
+      - master
```

Comment:
@joeaero19 Add the standard list of protected stages.  Any of the other services will have the full list in their serverless.yml
```    stages: # This is a list of common names for important envs that should not be destroyed.  You can remove the stage names your project doesn't use; this list is meant to be inclusive.
      - master
      - val
      - production
      - develop
      - main
      - impl
      - val
      - prod``

Reply:
done.